src/styles/card/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    margin: 32px 0 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-info {
+    text-align: start;
+    margin: 0 16px;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $black-color-for-text;
+    margin-bottom: 5px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 11px;
+    color: $grey-color-for-text;
+    margin-bottom: 18px;
+  }
+
+  &__price-list {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px ;
+  }
+
+  &__price {
+    font-size: 12px;
+    color: $grey-color-for-text;
+  }
+
+  &__amount {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+    color: $black-color-for-text;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    padding: 11px 0 ;
```

Comment:
```suggestion
    padding: 11px 0;
```

Reply:
Its something strange. It looks fine in VScode.